docs/core/getting-started/install/win-vs2017.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Install for Windows - Visual Studio 2017
+description: Install for Windows - Visual Studio 2017
+keywords: .NET, .NET Core, Visual Studio 2017, VS2017, C#, Getting Started, Acquisition, Cross Platform
+author: kendrahavens
+ms.author: kendrahavens
+ms.date: 12/19/2016
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 
+---
+# Install for Windows - Visual Studio 2017
+## 1. Download Visual Studio 2017 RC
+- An experimental release of .NET Core tools for Visual Studio is included in Visual Studio 2017 RC.
+  ## [Download Visual Studio 2017 RC](https://www.visualstudio.com/vs/visual-studio-2017-rc/)
+## 2. Install the .NET Core tools preview workload
+- Make sure you installed the workload ".NET Core and Docker (Preview)". You can check and install workloads by launching the Visual Studio Installer.
+
+  <img src="media/with-win-vs2017/filenewproject.png" width="800"/>
+
+## 3. Create a new .NET Core project
+- Click on File / New project and select the project template C# / .NET Core / .NET Core Console application.
+
+  <img src="media/with-win-vs2017/filenewproject.png" width="800"/>
+
+## 4. Add some code
+- Write some code in your Main method.
+
+  <img src="media/with-win-vs2017/helloworld.png" width="800"/>
+
+## 5. Run your application
+- Click on the menu item Debug / Start debugging to launch and debug your new .NET Core application from Visual Studio.
+
+  <img src="media/with-win-vs2017/debug.png" width="800"/>
+
+## And you're ready!
+You now have .NET core running on your machine!
```

Comment:
should we also change this topic: https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/getting-started/with-visual-studio-2017?branch=pr-en-us-1341 to not have install instructions in the prerequisites and instead just link to this topic?